api/src/runtime.js

```diff
@@ -8,20 +8,21 @@ const path = require('path');
 const runtimes = [];
 
 class Runtime {
-    constructor({ language, version, aliases, pkgdir, runtime }) {
+    constructor({ language, version, aliases, pkgdir, runtime, limit_overrides }) {
         this.language = language;
         this.version = version;
         this.aliases = aliases || [];
         this.pkgdir = pkgdir;
         this.runtime = runtime;
+        this.limit_overrides = limit_overrides;
     }
 
     static load_package(package_dir) {
         let info = JSON.parse(
             fss.read_file_sync(path.join(package_dir, 'pkg-info.json'))
         );
 
-        let { language, version, build_platform, aliases, provides } = info;
+        let { language, version, build_platform, aliases, provides, limit_overrides } = info;
         version = semver.parse(version);
 
```

Comment:
For each runtime, pre-calculate the exact limits for each runtime, should give faster runtime performance

Reply:
So we want to extract the limit_overrides properties directly into the runtime?

Reply:
pre-calculate them in the runtime - yes, because they can change per-runtime only

Reply:
Ok so if I understand correctly, in runtime.js we would decide whether to take the overrides from config.js or the package info instead of doing it in job.js.

Reply:
Correct

Reply:
And we would add a property in the runtime constructor for each limit?